### PR TITLE
SV: fix DC approximation

### PR DESCRIPTION
**Pull request by marqueslanauja**
**Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**

**What kind of change does this PR introduce?**

Feature

**What is the current behavior?**

Not all the otherSide methods support the Dc approximation.

**What is the new behavior (if this is a feature change)?**

Dc approximation has been added to all the otherSide methods

**Does this PR introduce a breaking change or deprecate an API?**

- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration guide has been updated in the github wiki *(What changes might users need to make in their application due to this PR?)*

**Other information**:

### CODE REVIEW

File: iidm/iidm-api/src/test/java/com/powsybl/iidm/network/util/SVTest.java [hunk 1]
```diff
@@ -414,11 +414,16 @@ void testDcLine() {
         SV svA2 = svA1.otherSide(line);
```
Comment (annetill) [inline iidm/iidm-api/src/test/java/com/powsybl/iidm/network/util/SVTest.java:414]: Can you add a unit test on a real network with a Boundary.getP() ?

Comment (marqueslanauja) [reply iidm/iidm-api/src/test/java/com/powsybl/iidm/network/util/SVTest.java:489]: Done. Adding the test i found a bug in the Dc approximation. The reactance must be used, expressed in per unit.

File: iidm/iidm-api/src/main/java/com/powsybl/iidm/network/util/SV.java [hunk 2]
```diff
@@ -262,8 +235,21 @@ private boolean isAllDataForCalculatingOtherSide() {
         return !(Double.isNaN(p) || Double.isNaN(q) || Double.isNaN(u) || Double.isNaN(a));
     }
 
-    private boolean isAllDataForCalculatingOtherSideDcApproximation() {
-        return !(Double.isNaN(p) || Double.isNaN(a));
+    private boolean isAllDataForCalculatingOtherSideDcApproximation(double zb) {
+        return !(Double.isNaN(p) || Double.isNaN(a) || Double.isNaN(zb));
+    }
+
+    private SV otherSide(double r, double x, double g1, double b1, double g2, double b2, double rho, double alpha, double zb) {
```
Comment (zamarrenolm) [inline iidm/iidm-api/src/main/java/com/powsybl/iidm/network/util/SV.java:242]: maybe we could rename the normalization parameter to v1v2 ?

**Review by zamarrenolm — APPROVED** [review]

File: iidm/iidm-api/src/main/java/com/powsybl/iidm/network/util/SV.java (hunk 2 above)
Comment (marqueslanauja) [reply iidm/iidm-api/src/main/java/com/powsybl/iidm/network/util/SV.java]: Done. I changed by zbase.
